WTWSMS/history/provinces/478 - Monemvasia.txt

```diff
@@ -4,13 +4,13 @@
 title = c_monemvasia
 
 # Settlements
-max_settlements = 4
+max_settlements = 6
```

Comment:
That land is very fertile and has huge potential for increase in population. It does not _start_ with those titles full.